docs/get-started/_index.md

```diff
@@ -16,32 +16,27 @@ images: ["/general/understand.png"]
-All these machines start by combining hardware and software.
-
 Viam is a software platform that makes it easy to combine and integrate hardware and software to build machines, connect them with the cloud, and make them smarter with machine learning.
 
 <img src="../about-viam.png" width="800px" alt="Viam architecture on device where viam-server uses the machine configuration to manage drivers for different hardware and in the cloud where the platform offer data management, ML, and extended functionality.">
 
-## `viam-server`
+## Support
 
-At the core of Viam is the open-source `viam-server` executable which runs on a computer and provides drivers for hardware and manages software and data for a machine.
-If you are working with microcontrollers, [`viam-micro-server`](/installation/) is a lightweight version of `viam-server` which can run on resource-limited embedded systems that cannot run the fully-featured `viam-server`.
+Viam runs on **microcontrollers** and **64-bit Linux OS, macOS, or Windows Subsystem for Linux**.
 
-To use Viam with a machine, you create a configuration specifying which hardware and software the machine consists of.
-Viam has many built-in {{< glossary_tooltip term_id="component" text="components" >}} and {{< glossary_tooltip term_id="service" text="services" >}} that run within `viam-server`.
-The components and services are configurable building blocks you can put together to make your machine.
-`viam-server` then manages and runs the drivers for the configured hardware components and software services.
+{{< board-carousel >}}
+<br>
 
-For example, if you are building a pet-feeding machine you might install `viam-micro-server` on a microcontroller and create a machine configuration with a camera and a servo for the hardware you are using.
+## Install Viam on your machine
 
-### Standard APIs
+At the core of Viam is the open-source `viam-server` executable which runs on a computer and provides drivers for hardware and manages software and data for a machine.
+If you are working with microcontrollers, [`viam-micro-server`](/installation/) is a lightweight version of `viam-server` which can run on resource-limited embedded systems.
 
-Each category of {{< glossary_tooltip term_id="resource" text="resource" >}} has a **standardized API** that you can access with an [SDK (software development kit)](/sdks/) in your preferred programming language.
-For example, you can send the same [`SetPower` command](/appendix/apis/components/motor/#setpower) to any kind of motor, using any of the available SDKs:
+- **Configuration**: create a configuration specifying which hardware {{< glossary_tooltip term_id="component" text="components" >}} and software {{< glossary_tooltip term_id="service" text="services" >}} the machine consists of and `viam-server` turns that into reality.
```

Comment:
"Turns that into reality" frankly sounds like hand wavy marketing speak. What do we really mean?
```suggestion
- **Configuration**: Create a configuration file specifying which hardware {{< glossary_tooltip term_id="component" text="components" >}} and software {{< glossary_tooltip term_id="service" text="services" >}} the machine consists of.
This configuration provides `viam-server` with the necessary information to access the components and services.
```

Reply:
```suggestion
- **Configuration**: Create a configuration file specifying which hardware {{< glossary_tooltip term_id="component" text="components" >}} and software {{< glossary_tooltip term_id="service" text="services" >}} the machine consists of and `viam-server` runs the hardware drivers and software to operate your machine.
```